src/rpc/blockchain.cpp

```diff
@@ -1137,6 +1137,8 @@ UniValue getblockchaininfo(const JSONRPCRequest& request)
             "  \"chainwork\": \"xxxx\"     (string) total amount of work in active chain, in hexadecimal\n"
             "  \"pruned\": xx,             (boolean) if the blocks are subject to pruning\n"
             "  \"pruneheight\": xxxxxx,    (numeric) lowest-height complete block stored\n"
+            "  \"disk_size\": xxxxxx,    (numeric) the estimated size of the block and undo files on disk\n"
```

Comment:
nit: perhaps `disk_usage` or `size_on_disk`?

Also consider placing this above `pruned` so all of the pruning-related fields are grouped.

Reply:
Also align description.

Reply:
IMO we should test the response result. In this case, at least, check that the key exists and value is a number. cc @jnewbery

Reply:
Ah, @jnewbery already recommended a test! 👍